README.md

```diff
@@ -341,6 +341,14 @@ There are certain environment variables used to configure server logging. The de
 
 All the **HTTP requests** get logged by default in the console during development.
 
+#### Mocking captive portal login and logout
+
+Captive portal login and logout can be mocked by using the endpoints
+mentioned in [openwisp-radius captive portal mock docs](https://openwisp-radius.readthedocs.io/en/latest/developer/captive_portal_mock.html).
+
+By default, it will be used in the development environment. These endpoints can
+be configured by changing them in the respective organization configuration file.
```

Comment:
Ideas for improvement of this text:

These URLs from OpenWISP RADIUS will be used by default in the development environment.
The captive portal login and logout URLs and their parameters can be changed by editing the YAML configuration files.